### PR TITLE
Initial work to enable CoreFx perf rolling runs on Linux.

### PR DESCRIPTION
**Pull request by jorive**

### CODE REVIEW

**Comment by jorive** [comment]
test ci please

**Comment by jorive** [comment]
test ci please

**Comment by jorive** [comment]
test ci please

**Comment by jorive** [comment]
@dotnet-bot test Ubuntu14.04 perf

**Comment by DrewScoggins** [comment]
test ci please

**Comment by jorive** [comment]
test ci please

**Comment by jorive** [comment]
test ci please

**Comment by jorive** [comment]
test ci please

**Comment by jorive** [comment]
test ci please

**Comment by jorive** [comment]
test ci please

**Comment by jorive** [comment]
test ci please

**Comment by jorive** [comment]
test ci please

File: perf.groovy [hunk 1]
```diff
@@ -69,32 +74,62 @@ def osShortName = ['Windows 10': 'win10',
...
+                        shell("python3.5 \"\${WORKSPACE}/Tools/Microsoft.BenchView.JSONFormat/tools/machinedata.py\"")
+                        shell("sudo -E bash ./build-managed.sh -release -tests -- /p:Performance=true /p:TargetOS=${osGroup} /m:1 /p:LogToBenchview=true /p:BenchviewRunType=${runType}")
+                    }
                 }
             }
 
             // Set the label.
-            newJob.with {
-                label('windows_clr_perf')
+            if (os == 'Windows_NT') {
```
Comment (DrewScoggins) [inline perf.groovy]: I think we can remove this.  Can you test it out and make sure?

**Comment by DrewScoggins** [comment]
Other than the one comment this LGTM

**Comment by jorive** [comment]
test ci please

**Pull request merged**